app/src/main/res/values-en/strings.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
 
     <string name="by_author_on_date">by %1$s, %2$s</string>
+  <string name="search_noun">Search</string>
```

Comment:
Don't add strings to english locale. English is the default locale.

Also pretty sure that 'search' is a verb in this context